src/xcode/ENA/ENA/Source/Services/PPAC/PPACService.swift

```diff
@@ -44,6 +47,13 @@ class PPACService: PrivacyPreservingAccessControl {
 		deviceCheck.deviceToken(apiToken.token, completion: completion)
 	}
 
+	#if !RELEASE
+	// needed to mak it possible to get called from rhe developer menu
```

Comment:
```suggestion
	// needed to mak it possible to get called from the developer menu
```

Reply:
will fix typo it on an other PR - thanks